modules/samples/artifacts/PublishHttpOAuthRequest/PublishHttpOAuthRequestWithOAuthUser.siddhi

```diff
@@ -0,0 +1,77 @@
+@App:name("PublishHttpOAuthRequestWithOAuthUser")
+@App:description("Send an OAuth events via HTTP transport with OAuth username and OAuth password")
```

Comment:
Here also remove 'OAuth events' part (we are sending HTTP events to an OAuth-protected endpoint). Also rather than saying 'with OAuth username and OAuth password', let's say the grant type as it's a standard that everyone knows.